src/schema/mutation/editUser.ts

```diff
@@ -41,6 +44,15 @@ export default {
                 path: 'roles',
                 match: { application: args.application } // Only returns roles attached to the application
             });
+
+            const positionAttributesNew = args.positionAttributes.filter(element => element.value.length > 0);
+            await User.findByIdAndUpdate(
+                args.id,
+                {
+                    positionAttributes: positionAttributesNew,
```

Comment:
@PacomeRivier ⚠️ will fix that, but that erases position attributes from other apps

Reply:
@AntoineRelief Indeed, there is also a display issue. It's stranged I missed this one because I remember testing this use case.